MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj

```diff
@@ -138,6 +139,9 @@
       <Name>MvvmCross.Droid.Support.Core.UI</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <AndroidResource Include="Resources\Xml\Empty.xml" />
```

Comment:
Without that file (or an empty `strings.xml` etc) Xamarin will not make `Resource.Layout.support_simple_spinner_dropdown_item` accessible.

You can't simply reference: `Android.Support.v7.AppCompat.Resource.Layout.support_simple_spinner_dropdown_item` in your code.  Instead Xamarin pulls those resources into your app's namespace during the build but only if you trick it into generating resources.